tests/xharness/Jenkins/TestSelector.cs

```diff
@@ -149,15 +168,21 @@ void SelectTestsByModifiedFiles (int pullRequest)
 			MainLog.WriteLine ("Found {0} modified file(s) in the pull request #{1}.", files.Count (), pullRequest);
 			foreach (var f in files)
 				MainLog.WriteLine ("    {0}", f);
+
+			var modifiedFiles = new Dictionary<string, (string [] Prefixes, TestSelection Selection)> {
+				["mtouch"] = (Prefixes: mtouchPrefixes, Selection: TestSelection.Mtouch),
```

Comment:
You're mapping string to enum here, which is already done elsewhere, which leaves room for typos.

It should be possible to make something like this work instead:

```csharp
var modifiedFiles = new Dictionary<TestSelection, string []> {
	[TestSelection.Mtouch] = mtouchPrefixes,
	...,
};
```